thirdparty/cmdconfig/commands/cmdtree/cmdtree_test.go

```diff
@@ -83,12 +83,12 @@ spec:
 		return
 	}
 
-	if !assert.Equal(t, `.
+	if !assert.Equal(t, fmt.Sprintf(`%q
```

Comment:
%q ? It should be %s correct if it is directory with no Kptfile

Reply:
Yes, will change that.